src/Workspaces/Core/Portable/Rename/ConflictEngine/DeclarationConflictHelpers.cs

```diff
@@ -58,51 +61,43 @@ public bool Equals(List<ITypeSymbol> x, List<ITypeSymbol> y)
                 return x.SequenceEqual(y);
```

Comment:
There is an [extension method](https://github.com/dotnet/corefx/blob/41e203011152581a6c65bb81ac44ec037140c1bb/src/System.Collections.Immutable/src/System/Linq/ImmutableArrayExtensions.cs#L168) as long as you're ```using System.Linq;```

In which case, you don't need to check the two lengths yourself because the extension method does that.